packages/augur-state/src/db/TrackedUsers.test.ts

```diff
@@ -0,0 +1,22 @@
+import { TrackedUsers } from "./TrackedUsers";
+
+
+class WrappedTrackedUsers extends TrackedUsers {
+    async wipe() {
+        await this.destroy();
+        this.db = this.makeDb();
+    }
+}
+
+
+test("track a user", async () => {
+    const trackedUsers = new WrappedTrackedUsers();
+    await trackedUsers.wipe();
```

Comment:
beforeEach? With just one test it shouldn't matter but I just got a new keyboard and need to practice typing on it. ;)

Reply:
Definitely! I'll put that in there.